src/Reflection/ReflectionMethod.php

```diff
@@ -245,7 +245,7 @@ public function isConstructor()
      *
      * @return bool
      */
-    public function isDestructor()
+    public function isDestructor() : bool
     {
         return $this->getName() === '__destruct';
```

Comment:
Created #264 tho, if you type your destructors like that, you should be slapped with a fish :D